test/testuninitvar.cpp

```diff
@@ -4248,7 +4248,7 @@ class TestUninitVar : public TestFixture {
                         "    int *x;\n"
                         "    int *&y = x;\n"
                         "}");
-        ASSERT_EQUALS("", errout.str());
+        TODO_ASSERT_EQUALS("", "[test.cpp:4]: (error) Uninitialized variable: x\n", errout.str());
```

Comment:
should there be a warning here? doesn't y just get a reference to x.

Reply:
This test seems incomplete. Actually, we do not warn when the variable is initialized through the reference:

```cpp
void foo() {
    int *x;
    int *&y = x;
    y = nullptr;
}
```

I can add `y = nullptr` to the test.